### DISCUSSION TITLE
Hello World!

### DISCUSSION THREAD

**Discussion by giscus** [aCursedComrade/acursedcomrade.github.io#4] [General]
# Hello World!

The birth of Comrade's Blog.

https://acursedcomrade.github.io/2022/12/hello/

**Comment by aCursedComrade** [comment]
The first comment! :rocket:

**Reply by aCursedComrade** [reply]
![Maxwell](https://media.discordapp.net/attachments/419288684014075904/1079933463513272413/D1604373-7A2B-41E8-A1C0-46661F9291F4.gif)

**Reply by gr4phite** [reply]
Maxwell got deleted :(

**Reply by aCursedComrade** [reply]
That's unfortunate but no need to worry, got him tucked away safely :)

![maxwell](https://github.com/aCursedComrade/acursedcomrade.github.io/assets/71869007/4bfa1bc8-8a97-4224-a27c-9efc05adcc31)
